content/en/gethelp.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Get Help
+sidebar: false
+---
+
+**User questions:** The best way to get help is to post your question to a site
+like [StackOverflow](http://stackoverflow.com/questions/tagged/scipy), with
+thousands of users available to answer.  Smaller alternatives include
+[IRC](https://webchat.freenode.net/?channels=%23scipy) and
+[Reddit](https://www.reddit.com/r/Scipy/). We wish we could keep an eye on
+these sites, or answer questions directly, but the volume is just a little
+overwhelming!
```

Comment:
We removed IRC recently and I guess we should remove reddit as well. Discourse would be better as we said elsewhere.
```suggestion
**User questions:** The best way to get help is to post your question to
the [user mailing list](https://mail.python.org/archives/list/scipy-user@python.org/)
or a site
like [StackOverflow](http://stackoverflow.com/questions/tagged/scipy), with
thousands of users available to answer. We wish we could keep an eye on
these sites, or answer questions directly, but the volume is just a little
overwhelming!
```